src/libs/ReportActionsUtils.ts

```diff
@@ -22,28 +22,28 @@ type LastVisibleMessage = {
     lastMessageHtml?: string;
 };
 
-type MessageActionItemChannelLogBase = {
+type MemberChangeMessageElementBase = {
     readonly kind: string;
     readonly content: string;
 };
 
-type MessageActionItemChannelLogText = {
+type MemberChangeMessageTextElement = {
     readonly kind: 'text';
     readonly content: string;
 };
 
-type MessageActionItemChannelLogUserMention = {
+type MemberChangeMessageUserMentionElement = {
     readonly kind: 'userMention';
     readonly accountID: number;
-} & MessageActionItemChannelLogBase;
+} & MemberChangeMessageElementBase;
 
-type MessageActionItemChannelLogRoomReference = {
+type MemberChangeMessageRoomReferenceElement = {
     readonly kind: 'roomReference';
     readonly roomName: string;
     readonly roomID: number;
-} & MessageActionItemChannelLogBase;
+} & MemberChangeMessageElementBase;
 
-type MessageActionItemChannelLog = MessageActionItemChannelLogText | MessageActionItemChannelLogUserMention | MessageActionItemChannelLogRoomReference;
+type MessageActionItemChannelLog = MemberChangeMessageTextElement | MemberChangeMessageUserMentionElement | MemberChangeMessageRoomReferenceElement;
```

Comment:
-> `MemberChangeMessageElement`

Maybe I missed this in my list, but I think this is quite clear